src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -428,12 +428,16 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
         //this is a nested list, each block with many baker rolls
         val writeBakers = TezosDb.writeVotingRolls(bakersBlocks)
 
-        val updateAccountsHistory = bakersBlocks.traverse {
+        val updateAccountsHistory = bakersBlocks.filterNot {
```

Comment:
if you look at my PR you'll see that this will be removed as I changed the logic a bit

Reply:
I already guessed that, but I kept things independent to avoid coupling separate PRs...